code/datums/status_effects/xeno_buffs.dm

```diff
@@ -281,14 +275,10 @@
 	var/datum/action/xeno_action/activable/essence_link/essence_link_action
 	/// References the Enhancement action and its vars.
 	var/datum/action/xeno_action/enhancement/enhancement_action
-	/// The plasma cost per tick of this ability.
-	var/plasma_cost
-	/// Damage bonus given by this ability.
-	var/damage_multiplier = 1.1
-	/// Speed bonus given by this ability.
-	var/speed_addition = -0.4
 	/// If the target xeno was within range.
 	var/was_within_range = TRUE
+	/// The plasma cost per tick of this ability.
+	var/plasma_cost
```

Comment:
You forgot to remove that

Reply:
Copypasting this from the earlier reviews:
To note, this var has its value setup later, so I don't really think it's viable to set it in the action instead.

Reply:
Hum okay, reasonable